src/pages/journal/journal.ts

```diff
@@ -22,7 +22,7 @@ export class JournalPage {
   ) {}
 
   ionViewDidLoad() {
-    this.journalProvider.getData('email').subscribe(data => {
+    this.journalProvider.getData('test@test.com').subscribe(data => {
```

Comment:
Need to be wary of using resolvable email addresses as dummies; use example.org email addresses.

AWS starts charging extra for bounced emails, I know that won't matter in this case but just something to keep in mind. Might be worth changing it here just for best practices